datadog_checks_dev/datadog_checks/dev/tooling/commands/ci/__init__.py

```diff
@@ -0,0 +1,22 @@
+# (C) Datadog, Inc. 2018-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import click
+
+from ..console import CONTEXT_SETTINGS
+from .setup import setup
+
+ALL_COMMANDS = (setup,)
+
+
+@click.group(context_settings=CONTEXT_SETTINGS, short_help='Collection of CI utilities')
+def ci():
```

Comment:
Note sure we creating a new `ci` namespace is the best way to go. Let me know if you have a better idea :)